pkg/odo/genericclioptions/runnable.go

```diff
@@ -189,7 +189,9 @@ func CheckMachineReadableOutputCommand(cmd *cobra.Command) error {
 
 	// Check the valid output
 	if hasFlagChanged && outputFlag.Value.String() != "json" {
+		//revive:disable:error-strings This is a top-level error message displayed as is to the end user
 		return fmt.Errorf("Please input a valid output format for -o, available format: json")
+		//revive:enable:error-strings
```

Comment:
I find it a little weird that you need to enable the lint after disabling it. Will adding the disabling comment inline not require enabling it again?

Reply:
I had the same feeling, but this is how `revive` recommends using comment directives to configure it: https://github.com/mgechev/revive#comment-directives.
If we don't enable it again, the `error-strings` rule will be disabled for the rest of the file.

Reply:
By the way, this makes me think that the `fmt.Errorf` here should be replaced by `errors.New` instead. I'll update it.